.vscode/settings.json

```diff
@@ -5,5 +5,8 @@
     { "language": "typescript", "autoFix": true },
     { "language": "typescriptreact", "autoFix": true }
   ],
-  "typescript.tsdk": "node_modules/typescript/lib"
-}
\ No newline at end of file
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
This file was also added to .gitignore so this can be removed.

Reply:
Removed the file.